bin/grep

```diff
@@ -92,35 +92,35 @@ sub VERSION_MESSAGE {
 
 sub usage {
 	die <<EOF;
-usage: $Me [-IincwsxvHhLlFguRraqT] [-e pattern] [-A NUM] [-B NUM] [-C NUM]
+usage: $Me [-acFgHhIiLlnqRrsTuvwxZ] [-e pattern] [-A NUM] [-B NUM] [-C NUM]
            [-m NUM] [-f pattern-file] [pattern] [file...]
 
 Options:
-	-i   case insensitive
-	-n   number lines
+	-A   show lines after each matching line
+	-a   treat binary files as plain text files
+	-B   show lines before each matching line
+	-C   show lines of context around each matching line
 	-c   give count of lines matching
-	-w   word boundaries only
-	-q   quiet; nothing is written to standard output
-	-x   exact matches only
-	-v   invert search sense (lines that DON'T match)
-	-H   show filenames
-	-h   hide filenames
 	-e   expression (for exprs beginning with -)
+	-F   search for fixed strings (disable regular expressions)
```

Comment:
thank you for alphabetizing this! I hate when these sorts of docs don't have an apparent sequence.